schemas/fingerprint-server-api.yaml

```diff
@@ -699,9 +697,57 @@ components:
                   description: >
                     Android specific emulator detection. There are 2 values:
                     • `true` - Emulated environment detected (e.g. launch inside of AVD)
-                    • `false` - No signs of emulated environment detected
-
-                    Available only for events from Android client. The field will not be present for a browser or iOS event.
+                    • `false` - No signs of emulated environment detected or the client isn't Android.
+                  example: false
+            clonedApp:
+              title: WebhookSignalResponseClonedApp
+              type: object
+              properties:
+                result:
+                  type: boolean
+                  description: >
+                    Android specific cloned application detection. There are 2 values:
+                    • `true` - Cloned application detected (e.g. fully cloned application with a slightly different package name)
+                    • `false` - No signs of cloned application detected or the client is not Android.
+                  example: false
+            factoryReset:
+              title: WebhookSignalResponseFactoryReset
+              type: object
+              properties:
+                time:
+                  title: Time
+                  description: >
+                    Time in UTC for the Android client when recent factory reset was done. 
+                    If there is no sign of factory reset or the client isn't Android, the field will be epoch time.
+                  type: string
+                  format: date-time
+                  example: '2022-06-09T22:58:36Z'
+                timestamp:
+                  description: Same value as it's in the `time` field but represented in timestamp format.
+                  type: integer
+                  format: int64
+                  example: 1654815517198
+            jailbroken:
+              title: WebhookSignalResponseJailbroken
+              type: object
+              properties:
+                result:
+                  type: boolean
+                  description: >
+                    iOS specific jailbreak detection. There are 2 values:
+                    • `true` - Jailbreak detected
+                    • `false` - No signs of jailbreak or the client is not iOS.
+                  example: false
+            frida:
+              title: WebhookSignalResponseFrida
+              type: object
+              properties:
+                result:
+                  type: boolean
+                  description: >
+                    iOS specific frida detection. There are 2 values:
```

Comment:
Can we turn `Frida` into a link like https://frida.re/docs/ios/ ?

Reply:
done